split_hf_unlabelled_into_multiple_parts.py

```diff
@@ -0,0 +1,41 @@
+"""
+python split_hf_unlabelled_into_multiple_parts.py
+
+"""
```

Comment:
Needs clearer explanation of what it does - i.e this is the situation you run it in, this is what it produces, then you can upload to HF like this.

Reply:
Done